plugins/woocommerce/includes/admin/settings/class-wc-settings-page.php

```diff
@@ -252,6 +248,20 @@ protected function get_section_settings_data( $section_id, $sections ) {
 			return $section_settings_data;
 		}
 
+		/**
+		 * Get the custom view object.
+		 *
+		 * @param string $content The content of the custom view.
+		 * @return array The custom view object.
+		 */
+		public function get_custom_view_object( $content ) {
```

Comment:
nit: I'm not sure if it will be used by other classes. If it doesn't, we can convert it to a private or protected method.